app/experimenter/experiments/constants/nimbus.py

```diff
@@ -448,12 +448,26 @@ class NimbusTargetingConfig:
         !('media.videocontrols.picture-in-picture.video-toggle.has-used'|preferenceValue)
         &&
         ('media.videocontrols.picture-in-picture.enabled'|preferenceValue)
+        &&
+        ('media.videocontrols.picture-in-picture.video-toggle.enabled'|preferenceValue)
     )
     """,
     desktop_telemetry="",
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_PIP_NEVER_USED_STICKY = NimbusTargetingConfig(
+    name="PiP Never Used (Sticky)",
+    slug="pip_never_used_sticky",
+    description="Users that have never used Picture in Picture, with sticky enrollment",
+    targeting="(({pip}) || ({sticky}))".format(
+        pip=TARGETING_PIP_NEVER_USED.targeting,
```

Comment:
My understanding here would be that `TARGETING_PIP_NEVER_USED` should point it's targeting to `TARGETING_PIP_NEVER_USED_STICKY` (here `_STICKY` points to the old never used config). That way we don't need to change any references to the old targeting config but they would still get sticky 🤔 

Or is this just adding a new targeting config that can be used for future stuff?

Reply:
Well, we could add in the sticky config to the old one, update it in place, except if existing experiments point to it then they'll be using a config whose behaviour is different than they expected to be when they chose it.  So instead we're adding a new one that adds the sticky behaviour next to the previous ones config.  

I'm glossing over a bit about keeping the old one intact since we're also adding a third pref to it but my understanding from the PIP team is that that pref should've always been present in the first one for it to perform the filtering they expected from it.

Reply:
Totally makes sense 👍